proposals/lip-0054.md

```diff
@@ -100,24 +100,24 @@ Since these technical requirements are not straightforward, the recovery command
 | Name                                          | Type   | Value        |
 |-----------------------------------------------|--------|--------------|
 | **Module**                                    |        |              |
-| `MODULE_ID_INTEROPERABILITY`                  | uint32 | 64           |
-| `MODULE_ID_TOKEN`                             | uint32 | TBD          |
+| `MODULE_ID_INTEROPERABILITY`                  | bytes  | TBD          |
```

Comment:
comment for line 24 above: "the users of *the* said chain".

Reply:
I think you can omit the article here.